core/src/main/java/org/mule/runtime/core/internal/policy/CompositeSourcePolicy.java

```diff
@@ -125,7 +125,11 @@ public CompositeSourcePolicy(List<Policy> parameterizedPolicies,
    */
   @Override
   protected Publisher<CoreEvent> processPolicy(Policy policy, Processor nextProcessor, CoreEvent event) {
-    return just(event).transform(sourcePolicyProcessorFactory.createSourcePolicy(policy, nextProcessor));
+    return just(event)
+        .doOnNext(s -> logPolicy(getCoreEventId(event), getPolicyName(policy), getCoreEventAttributesAsString(event)))
```

Comment:
Should have different messages for the start and the end of the policy execution. Right now is always just showing `Processing`